sql/src/main/java/io/crate/metadata/table/AbstractTableInfo.java

```diff
@@ -101,6 +106,11 @@ public TableParameterInfo tableParameterInfo() {
         return ImmutableMap.of();
     }
 
+    @Nullable
+    public String getAnalyzerForColumnIdent(ColumnIdent ident) {
```

Comment:
can be removed if the resolver requires DocTableInfo instead of AbstractTableInfo